.gitmodules

```diff
@@ -0,0 +1,4 @@
+[submodule "ceph-nvmeof-spdk"]
+	path = ceph-nvmeof-spdk
+	url = https://github.com/sdpeters/ceph-nvmeof-spdk.git
+    branch = wip-ceph-nvmeof
```

Comment:
Nit: align with previous lines (replace spaces with a tab) and add a newline at the end.

Reply:
I'll fix that